frontend/styles.css

```diff
@@ -17,23 +38,57 @@ img {
     margin-right: auto;
 }
 
-h3 {
-    text-align: center;
-}
-
-.form-input {
-    padding: 10px
+a {
+    color: white;
 }
 
+/* login page */
 .login {
     width: 50%;
     display: block;
     margin-left: auto;
     margin-right: auto;
-    border: 2px solid grey;
-    background-color: lightgrey;
+    background-color: #E0E8ED;
+    border-radius: 1em;
+    padding: 30px;
+    margin-top: 100px;
+}
+
+.login h2 {
+    text-align: center;
+    font-size: 120%;
+    padding: 10px;
+}
+
+.form-input {
+    padding: 10px;
+    margin: 10px;
+}
+
+.form-input input[type="text"], [type="password"]{
+    border: none;
+    background: transparent;
+    border-bottom: solid 1px;
+}
+
+.form-input input[type="submit"]{
+    width: 100%;
+    padding: 10px;
+    font-size: 150%;
+    border-radius: 0.5em;
+    background-color: #031D40;
+    color: #E0E8ED;
+    border: none;
+    box-shadow:  10px 10px 15px 0px #8B8E90;
+}
+
+.form-input input[type="submit"]:hover {
+    cursor: pointer;
+    background-color: #23551b;
+    border: solid 3px #031D40;
+    color: #E0E8ED;
 }
 
-button {
-    margin: 20px;
+.login p a {
+    color: #031D40;
 }
```

Comment:
Did you know that CSS also has constants?

https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties

If you're going to switch to sass at some point (which I would want to as well) then there's not really any need, but it's nice that it's there.

Reply:
That's really good to know. That's why I wanted to use SASS: to introduce constants.

Reply:
Just added CSS variables.